packages/cli/src/commands/deploy.ts

```diff
@@ -291,7 +300,16 @@ const selectApiScope = async (token: string): Promise<Scope> => {
     scope: 'partner-portal',
   }).isAuthenticated()
 
-  if (!accessibleToMyPlugins && !accessibleToPartnerPortal) {
+  const accessibleToOrganizationPortal = await StoryblokClient({
+    token,
+    scope: 'organization',
+  }).isAuthenticated()
+
+  if (
+    !accessibleToMyPlugins &&
+    !accessibleToPartnerPortal &&
+    !accessibleToOrganizationPortal
+  ) {
     console.error(
       red('[ERROR]'),
       `The token appears to be invalid as it does not have access to either My Plugins or the plugins on the Partner Portal.`,
```

Comment:
We need to include the organization portal here! :)

Reply:
true :D haha let me quickly add it